src/app/tests/suites/certification/ci-pics-values

```diff
@@ -2174,116 +2174,6 @@ ACFREMON.S.A0002=1
 ACFREMON.S.A0003=1
 ACFREMON.S.C00.Rsp=0
 
-# Ceramic Filter Monitoring Cluster
```

Comment:
can we just delete all the associated test files with these too?

Reply:
well, yes we should remove them. but I don't really want to do this in this PR, as there is this PR from @p0fi https://github.com/project-chip/connectedhomeip/pull/27893 . 
But i may have to do it anyways because the of the one job that is still failing (`Tests / Test Suites - Linux (no-ble-tsan-clang)` )

Reply:
ended up deleting the files